tests/integration/security/ca_custom_root/secure_naming_test.go

```diff
@@ -187,17 +184,12 @@ func TestSecureNaming(t *testing.T) {
 									PortName: "http",
 									Scheme:   scheme.HTTP,
 									Count:    callCount,
+									Validator: echo.And(
+										echo.ExpectOK(),
+										echo.ExpectReachedClusters(bSet.Clusters()),
+									),
 								}
-								checker := connection.Checker{
-									From:          a,
-									Options:       callOptions,
-									ExpectSuccess: tc.expectSuccess,
-									DestClusters:  bSet.Clusters(),
-								}
-								if err := retry.UntilSuccess(
-									checker.Check, retry.Delay(time.Second), retry.Timeout(15*time.Second), retry.Converge(5)); err != nil {
-									t.Fatal(err)
-								}
+								a.CallWithRetryOrFail(t, callOptions, retry.Converge(5))
```

Comment:
nit: the default is now converge(3) so we probably do not need converge(5)

Reply:
It's actually not an echo default currently. I think it probably should be though. Let me create a PR to change it.